sanctions/apps/api/v1/urls.py

```diff
@@ -1,4 +1,13 @@
 """ API v1 URLs. """
+from django.urls import re_path
+
+from sanctions.apps.api.v1.views import SanctionsCheckView
 
 app_name = 'v1'
 urlpatterns = []
+
+SDN_URLS = [
+    re_path(r'^sanctions-check/$', SanctionsCheckView.as_view(), name='sanctions-check'),
```

Comment:
this view and URL should probably also be renamed to SDN if we're specifically checking SDN

this may elicit the question "what?? then why did we name it sanctions service?" and i think the answer i would have is "if there are _other_ sanction related checks we need, we can put them here, but there's nothing wrong with calling stuff SDN within the sanctions service if it indeed is SDN related"